frontend/test/metabase/lib/expressions/compile.unit.spec.js

```diff
@@ -6,7 +6,7 @@ import {
   expressionOpts,
 } from "./__support__/expressions";
 
-const ENABLE_PERF_TESTS = !process.env["CI"];
+const ENABLE_PERF_TESTS = false; //!process.env["CI"];
```

Comment:
Were you just stopping these from running locally temporarily or do you want to remove them?